Firestore/core/src/firebase/firestore/model/snapshot_version.h

```diff
@@ -38,6 +43,29 @@ class SnapshotVersion {
   /** Creates a new version that is smaller than all other versions. */
   static const SnapshotVersion& None();
 
+#if defined(__OBJC__)
+  SnapshotVersion(FSTSnapshotVersion* version) {  // NOLINT(runtime/explicit)
+    if ([version isEqual:[FSTSnapshotVersion noVersion]]) {
+      timestamp_ = Timestamp{};
```

Comment:
I think this is unnecessary, as the timestamp_ instance will already be initialized with it's default ctor. (No objections if you want to keep it though.)